std/format/read.d

```diff
@@ -1,35 +1,69 @@
 // Written in the D programming language.
 
 /**
-   This is a submodule of $(MREF std, format).
-   It provides some helpful tools.
-
-   Copyright: Copyright The D Language Foundation 2000-2013.
-
-   License: $(HTTP boost.org/LICENSE_1_0.txt, Boost License 1.0).
-
-   Authors: $(HTTP walterbright.com, Walter Bright), $(HTTP erdani.com,
-   Andrei Alexandrescu), and Kenji Hara
-
-   Source: $(PHOBOSSRC std/format/read.d)
+This is a submodule of $(MREF std, format).
+
+It provides two functions for reading formatted $(LREF unformatValue)
+reads a single value, while $(LREF formattedRead) reads several
+values at once and matches the characters found between format
+specifiers.
+
```

Comment:
Suggested rewording:

"It provides two functions for reading formatted input: $(LREF unformatValue) and $(LREF formattedRead).
The former reads a single value. The latter reads several values at once and matches the characters found
between format specifiers."